upt_macports/templates/python.Portfile

```diff
@@ -18,6 +18,7 @@ distname            ${python.rootname}-${version}
 {% block versions %}
 python.versions     37
 
+## setuptools dependency is added by default and should be double checked
```

Comment:
this is not very clear or complete (see the Issue #15)... perhaps something like:
```
## setuptools is added by default as a build-dependency since this is often the case.
## However, if the package installs a "console_script" or uses "entry_points", setuptools
## should be added as a lib-dependency instead. Please verify from the setup.{py/cfg}
## file that setuptools is indeed used and that the dependency-type is correct.
```

and indeed move it to be just above the depedency.

Reply:
I agree that my initial comment is not complete as I am not so sure about adding so much in comments as this makes it hard to see the actual useful part, so just an idea maybe we can add all this in the output before the portfile or maybe just add it in the Readme as having it in all the Portfile looks a bit messy and as @mojca said at the end most of the comments should be deleted by the maintainer, so adding this means more work for the maintainers although I would agree that deleting comments is not time-consuming but I think it could be avoided.

Reply:
I think it's important to be complete here and in my opinion the whole comment if useful ;) Again, this comments will/should only be added when generating a new Portfile. It's not that hard to read through four lines of comments and know what you're supposed to do as a maintainer (also, removing 4 lines isn't much harder than removing 1).

I am not sure what you mean by "add all this in the output before the portfile"? I am against the idea of adding stuff into a Readme file, despite the name nobody is actually going to read that and comments should be close to where they actually apply.

Reply:
> I am not sure what you mean by "add all this in the output before the portfile"? I am against the idea of adding stuff into a Readme file, despite the name nobody is actually going to read that and comments should be close to where they actually apply.

Haha that's true no one reads the "Readme".
I meant that we can get rid of the whole comment and not even keep one line as it was in my previous commit and instead when we run upt it will print the "disclaimer/info" comments before the actual upt file(this will ofc be gone once we output to file). But I see that then it would be harder to get that which comment is referring to which line.